pkgs/applications/audio/quodlibet/default.nix

```diff
@@ -23,20 +22,22 @@
 , libsoup
 , webkitgtk
 
-# optional features
+  # optional features
 , withDbusPython ? false
-, withPypresence ? false
-, withPyInotify ? false
 , withMusicBrainzNgs ? false
 , withPahoMqtt ? false
+, withPyInotify ? false
+, withPypresence ? false
 , withSoco ? false
 
-# backends
-, withGstreamerBackend ? true, gst_all_1
+  # backends
+, gst_all_1
 , withGstPlugins ? withGstreamerBackend
-, withXineBackend ? true, xine-lib
+, withGstreamerBackend ? true
+, withXineBackend ? true
+, xine-lib
```

Comment:
`gst_all_1` and `xine-lib` as optional dependencies probably should be ordered consistently either before or after their corresponding flags, not alphabetically with all the flags.